### DISCUSSION TITLE
How can I set noice.nvim to defaults?

### DISCUSSION THREAD

**Discussion by t4Linux** [LazyVim/LazyVim#2551] [Q&A] [answered]
LazyVim defaults for noice.nvim makes it appear on top of the screen and I would like to set it to defaults - middle of the screan. I'm not so good with configuration so can someone please... helpm me to set it to the middle of the screen?
Thank you!

**Comment by zvrkSam** [comment, marked as answer]
This is what works for me, on my main monitor, you are going to have to tweak the row fields for yourself.
```lua
opts = {
    views = {
      cmdline_popup = {
        position = {
          row = 35,
          col = "50%",
        },
      },
      cmdline_popupmenu = {
        position = {
          row = 38,
          col = "50%",
        },
      },
    },
  }
  ```

**Reply by dpetka2001** [reply]
You can just use percentages instead of specific row. Something like `row = "50%"` for `cmdline_popup` and `row = "67%"` for `cmdline_popupmenu`. This way it should be more consistent.

**Reply by zvrkSam** [reply]
I have tried using % values, but when I hit tab for suggestions `cmdline_popupmenu` goes over `cmdline_popup` and blocks it for me.

**Comment by t4Linux** [comment]
I've added the settings but it still apears on top.

![2024-02-14_13:34:51](https://github.com/LazyVim/LazyVim/assets/125494464/bd58b77c-f8d1-439e-976e-69b026a59b72)

**Reply by zvrkSam** [reply]
From this image it seems that you are passing `opts = {}` two times, remove one.

**Comment by t4Linux** [comment]
Thank you VERY much. You don't know how much it bothered me. Once more thank you :)
